README.md

```diff
@@ -423,6 +423,14 @@ policyResult.Result - if executing a func, the result if the call succeeded or t
 */
 ```
 
+
+# Thread safety
+
+The existing policies (retry and circuit-breaker) are fully thread-safe, and the intention is that the forthcoming wider resilience policies will be thread-safe too. 
+
+While the internal operation of the policy is thread-safe, but this does not magically make delegates you execute through the policy thread-safe: if delegates you execute through the policy are not thread-safe, they remain not thread-safe.
```

Comment:
Thanks for this @kbilsted . Suggest just remove 'but' from this sentence for grammatical correctness.

@joelhulen looks like a good clarification from @kbilsted to merge!